### ISSUE TITLE
Problem importing pool in second-boot-bookworm-Debian.yml

### ISSUE THREAD

**Issue by HankB** [HankB/polana-ansible#9]
Two issues, possibly related. First, a previous pool is listed and can actually be imported! Second, the pool created by `provision-Debian.yml` lists unsupported features.

```text
root@bristol:~# zpool import
   pool: TW0M3TJT
     id: 8033565518440443361
  state: ONLINE
status: The pool was last accessed by another system.
 action: The pool can be imported using its name or numeric identifier and
        the '-f' flag.
   see: https://openzfs.github.io/openzfs-docs/msg/ZFS-8000-EY
 config:

        TW0M3TJT                           ONLINE
          nvme-eui.38f601563152399d-part3  ONLINE

   pool: higgs_tank
     id: 16736013408069462014
  state: UNAVAIL
status: The pool uses the following feature(s) not supported on this system:
        com.klarasystems:vdev_zaps_v2
action: The pool cannot be imported. Access the pool on a system that supports
        the required feature(s), or recreate the pool from backup.
 config:

        higgs_tank                    UNAVAIL  unsupported feature(s)
          mmc-JC1S5_0xd0c363ce-part3  ONLINE
root@bristol:~#
```

Ah... Never mind about `TW0M3TJT` as it is a pool on the NVME SSD currently installed in this host (which booted form an SD card.)

The problem with ` com.klarasystems:vdev_zaps_v2` remains.

**Comment by HankB** [comment]
It appears that `-o compatibility=...` should fix this. See

* <https://openzfs.github.io/openzfs-docs/man/master/8/zpool-create.8.html>
* <https://openzfs.github.io/openzfs-docs/man/master/7/zpool-features.7.html#Compatibility_feature_sets>

Feature sets are listed in `/usr/share/zfs/compatibility.d` and for this usage, it seems that `openzfs-2.1-linux` should work.  `2.1.11` is in Debian Bookworm. (2.2.2 is used in RpiOS which was used to run this playbook.)

**Comment by HankB** [comment]
Fixed with #a9b2f91deac00b01b9aa22f85941422a44461131
